src/kirby/components/toolbar/toolbar.component.tns.html

```diff
@@ -0,0 +1,27 @@
+<ActionBar class="toolbar {{themeColor}}">
+  <NavigationButton  icon="~/assets/images/arrow-back.png" [visibility]="showBackButton && isAndroid() ? 'visible' : 'collapsed'" (tap)="onBackButtonSelected()">
+  </NavigationButton>
+   <ActionItem *ngIf="showBackButton && !isAndroid()" ios.position="left" >
+    <StackLayout orientation="horizontal" class="toolbar-item-secondary">
+      <kirby-icon name="arrow-back" size="md"></kirby-icon>
+    </StackLayout>
+  </ActionItem>
+  <StackLayout orientation="horizontal"
+    ios:horizontalAlignment="center"
```

Comment:
If `horizontalAlignment` for both iOS and Android is `center`, is there need for two attributes then?
using `horizontalAlignment="center"` would do the same.